org.zotero.Zotero.yaml

```diff
@@ -10,13 +10,8 @@ finish-args:
   - --share=ipc
   - --share=network
   - --filesystem=home
```

Comment:
Maybe we should add a comment here with a link to #82 to make clear why we keep the `home` permission.